CHANGELOG.md

```diff
@@ -10,7 +10,131 @@ tries to adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 
-- Started keeping a changelog!
+
+## [0.44.12] - 2025-04-25
+
+### Added
+
+- [#2039](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2039) Add generic `shift_left!` and `shift_right!` for univariate polynomials
+- [#2047](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2047) Add `falling_factorial`
+
+### Changed
+
+- [#2055](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2055) Add comments and more solving examples
+- [#2058](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2058) Improve univariate conversion
+- [#2057](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2057) Fix a JET issue in `iterate(::ProductIterator)`
+- [#2056](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2056) Fix typo in `isomorphism(::Type{T}, ::T) where T <: Group` code
+- [#2054](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2054) Fix stack overflow when constructing non-implemented views
+- [#2048](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2048) Enable point views of matrices
+
+## [0.44.11] - 2025-03-28
+
+### Added
+
+### Changed
+
+- [#2043](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2043) Add an `@id` field to a docs header
```

Comment:
This PR currently does not honour PR labels. e.g. this one here has a `release notes: not needed` label

Reply:
This PR is manually created for now, as we don't have the machinery in place yet and I did not want to go through all PRs and set labels (far too much work). I manually removed a few PRs from the list, and tweaked the description of some. If you'd like to not mention a few changes, please make code change suggestions deleting them. 

Once @aaruni96 is back to work we can apply the release notes machinery we developed for Oscar also to AA and Nemo so that future updates can make use of the labels and automation. (And then we need to carefully sync improvements between the three -- one option for that would be to host the scripts in a separate repository.)